src/main/kotlin/uk/gov/justice/digital/hmpps/dpssmoketest/config/ResourceServerConfig.kt

```diff
@@ -18,7 +18,7 @@ class ResourceServerConfig : WebSecurityConfigurerAdapter() {
       sessionManagement { SessionCreationPolicy.STATELESS }
       csrf { disable() }
       authorizeRequests {
-        listOf("/webjars/**", "/favicon.ico", "/health/**", "/info", "/v3/api-docs/**", "/swagger-ui/**", "/swagger-ui.html",)
+        listOf("/webjars/**", "/favicon.ico", "/health/**", "/info", "/v3/api-docs/**", "/swagger-ui/**", "/swagger-ui.html", "/smoke-test-unsecured")
```

Comment:
This endpoint exists to manually test locally form a Chrome browser - it's not for the Circle experiment.